lib/cluster/cpg.c

```diff
@@ -457,44 +461,34 @@ pcmk__cpg_message_data(cpg_handle_t handle, uint32_t sender_id, uint32_t pid,
 
     if (msg->is_compressed && (msg->size > 0)) {
         int rc = BZ_OK;
-        char *uncompressed = NULL;
         unsigned int new_size = msg->size + 1;
+        char *uncompressed = pcmk__assert_alloc(1, new_size);
 
-
-        crm_trace("Decompressing message data");
-        uncompressed = pcmk__assert_alloc(1, new_size);
         rc = BZ2_bzBuffToBuffDecompress(uncompressed, &new_size, msg->data,
                                         msg->compressed_size, 1, 0);
-
         rc = pcmk__bzlib2rc(rc);
         if ((rc == pcmk_rc_ok) && (msg->size != new_size)) { // libbz2 bug?
             rc = pcmk_rc_compression;
         }
         if (rc != pcmk_rc_ok) {
-            crm_err("Decompression failed: %s " QB_XS " rc=%d",
-                    pcmk_rc_str(rc), rc);
             free(uncompressed);
-            goto badmsg;
+            crm_warn("Ignoring compressed CPG message %d from %s (ID %" PRIu32
+                    " PID %" PRIu32 "): %s",
```

Comment:
same indentation thing from the other PR, but not worth fixing for a backport

Reply:
ah that's a different one we actually missed in the original